src/sentry/tasks/digests.py

```diff
@@ -64,7 +66,9 @@ def deliver_digest(key, schedule_timestamp=None):
                 target_identifier,
                 fallthrough_choice=fallthrough_choice,
             )
+            metrics.incr("tasks.digest.deliver_digest", tags={"success": True})
         else:
+            metrics.incr("tasks.digest.deliver_digest", tags={"success": False})
             logger.info(
                 "Skipped digest delivery due to empty digest",
```

Comment:
asked dan about this line, it seems to happen if the issue is deleted when we go to send the digest